source/developers/projects/engine/api/site/cache/statistics.rst

```diff
@@ -42,18 +42,17 @@ Response
 
 .. code-block:: json
 
-  [
-    {
-      "46b0f2effb636293234e10749dd9f5a4": {
-        "size": 147,
-        "localHeapSize": 147,
-        "localOffHeapSize": 0,
-        "localDiskSize": 0,
-        "remoteSize": 0,
-        "writerQueueLength": 0,
-      }
-    }
-  ]
+  {
+    "size": 17,
+    "localHeapSize": 17,
+    "localOffHeapSize": 0,
+    "localDiskSize": 0,
+    "cacheHitRatio": 1,
```

Comment:
What's the ratio's unit? Should it be a fraction of sorts?

Reply:
the Ehcache API returns a `double ` for this field, keep in mind this in my local with a single session so the cache is barely used